marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/StepImpl.java

```diff
@@ -177,9 +174,20 @@ public void incrementVersion() {
         setVersion(getVersion() + 1);
     }
 
+    @Override
+    public void setId(String id) {
```

Comment:
Don't need ID field as stepId == stepName-stepType
That is how we search for the Step in the filesystem.